lib/PaymentIntent.php

```diff
@@ -23,23 +23,23 @@
  * @property string|null $customer
  * @property string|null $description
  * @property string|null $invoice
- * @property mixed|null $last_payment_error
+ * @property \Stripe\StripeObject|null $last_payment_error
```

Comment:
@ob-stripe Shouldn't this be an `\Stripe\ErrorObject` type? That's how we always use it.

Reply:
@ruudk It is. We're currently switching to a a new "codegen" tool that automatically generates PHPDoc type annotations based on the OpenAPI specification for Stripe's API. The tool didn't know about `\Stripe\ErrorObject`, but that will be fixed very soon (it's already been fixed internally, we just need to update the public repository). Thanks for the report!